packages/controllers/src/snaps/SnapController.test.ts

```diff
@@ -67,6 +67,9 @@ jest.mock('./utils/npm', () => ({
 
 fetchMock.enableMocks();
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const omitSourceCode = ({ sourceCode, ...rest }: any): any => rest;
```

Comment:
Rather than removing the source code from the result of `getSnapObject`, shouldn't we just remove the source code in that function directly, since the `Snap` object won't have it anymore?

Reply:
Seems like a better solution to create a separate function for it.

Reply:
Done!